src/eureka/S5_lightcurve_fitting/__init__.py

```diff
@@ -1,7 +1,23 @@
 """
 Package to fit models to light curve data
 """
+# Do some checks here to avoid spamming these warnings
+try:
+    import theano
+    import pymc3
+    import starry
+    import pymc3_ext
+    import arviz
+    success = True
+except ImportError:
+    success = False
+    print("Could not import starry and/or pymc3 related packages. "
+          "Functionality may be limited.")
+if success:
+    from . import differentiable_models
```

Comment:
I hesitated to catch any and all exceptions with importing this as it might hide some other issue. Looking more carefully at what throws `ImportError`s though, it seems it only gets thrown if "Import can't find module, or can't find name in module." which should only catch cases when one of the dependencies isn't installed (and not some other issue with our code). So in summary, I will do as you suggested.